version.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# Copyright (c) 2018 Anton Semjonov
+# Licensed under the MIT License
+
+#-------------------------------------------#
+#                                           #
+#  This script prints version information   #
+#         when executed in a shell:         #
+#                                           #
+#             sh ./version.sh               #
+#                                           #
+#         For more information see:         #
+#   https://github.com/ansemjo/version.sh   #
+#                                           #
+#-------------------------------------------#
+
+# Ignore warnings about necessary single-quotes and literal newlines for POSIX compatability:
```

Comment:
Does it work with non-bash sh, such as dash/ash or busybox sh?

Reply:
Or, to elaborate: the comment makes shellcheck ignore the use of a `$...` variable inside single-quotes, which is necessary so it is *not* expanded, and the literal newline after a backslash was one of the possibilities to have a portable way to match newlines with `sed`.

Reply:
I've added a few [Travis tests](https://travis-ci.com/ansemjo/version.sh) now and it works fine in `bash`, `dash`/`ash` and `ksh` on Linux as well as `bash` and `ksh` on OSX.